Blog-projects/Widget-window-in-Qt-Quick-app/widgetFormHandler.h

```diff
@@ -0,0 +1,37 @@
+#ifndef WIDGETFORMHANDLER_H
+#define WIDGETFORMHANDLER_H
+
+#include "fontcontrolswidgetsform.h"
```

Comment:
isn't `class FontControlsWidgetsForm;` enough in the header?
(and of course, do the real `#include` in the .cpp file)